linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -418,17 +418,17 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 ### threadpool_destroy_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, threadpool_work_item);
+static void threadpool_destroy_work_item(THANDLE(THREADPOOL_WORK_ITEM), threadpool_work_item);
```

Comment:
>threadpool_destroy_work_item [](http://example.com/codeflow?start=12&length=28)

Like in win32, should be named threadpool_dispose_work_item

Reply:
Done